src/cli.c

```diff
@@ -1000,3 +1030,21 @@ decrypt(SSL* ssl, int socket, char* path)
 
    return ret;
 }
+
+static int
+encrypt(SSL* ssl, int socket, char* path)
+{
+   int ret;
+   int number_of_returns = 0;
+   int code = 0;
+
+   ret = pgmoneta_management_encrypt(ssl, socket, path);
+   pgmoneta_management_read_int32(ssl, socket, &number_of_returns);
```

Comment:
I mimicked this piece of code based on decrypt. The `number_of_returns` is 1 on error, but why `number_of_returns` here? Both `decrypt` and `encrypt` should have only one int return value to indicate the exec result. 
The return value of `pgmoneta_decrypt_archive` indicates that we correctly sent the command to the server. But what about the actual exec return value? Which one should be the `$?`

Reply:
Yeah, that is an oversight in that commit. You can do

```
   int ret;

   if (pgmoneta_management_decrypt(ssl, socket, path))
   {
       return 1;
   }
   pgmoneta_management_read_int32(ssl, socket, &ret);
   return ret;
```

So, `0` upon successful decryption, and `1` upon failure